File: competitive-events-api/src/main/java/es/jocasolo/competitiveeventsapi/dto/comment/CommentPostDTO.java
package es.jocasolo.competitiveeventsapi.dto.comment;

import java.io.Serializable;

import javax.validation.constraints.NotBlank;

import es.jocasolo.competitiveeventsapi.dto.DTO;

public class CommentPostDTO extends DTO implements Serializable {

	private static final long serialVersionUID = 1L;

	private String text;

	@NotBlank
	private String eventId;

	public String getEventId() {
		return eventId;
	}

	public void setEventId(String eventId) {
		this.eventId = eventId;
	}

	public String getText() {
		return text;
	}

	public void setText(String text) {
		this.text = text;
	}

	@Override
	public String toString() {
		return "CommentPostDTO [text=" + text + "]";
	}

}

File: competitive-events-app/app/src/main/java/es/jocasolo/competitiveeventsapp/dto/comment/CommentPostDTO.kt
package es.jocasolo.competitiveeventsapp.dto.comment

import com.google.gson.annotations.SerializedName
import es.jocasolo.competitiveeventsapp.dto.HistoryItemDTO
import es.jocasolo.competitiveeventsapp.dto.user.UserDTO
import java.util.*

class CommentPostDTO(
        @SerializedName("text") var text: String,
        @SerializedName("eventId") var eventId: String
) : HistoryItemDTO() {
}